MosipNexus/Server/run_update.py

```
@@ -263,4 +263,15 @@ def run() -> None:
-    run()
+    import traceback
+    try:
+        run()
+    except Exception as exc:
+        error_detail = traceback.format_exc()
+        print(f"\nERROR: {exc}", flush=True)
+        try:
+            from notifications.email_notifier import send_job_failure_alert
+            send_job_failure_alert("nexus-updater", error_detail)
+        except Exception:
+            pass  # don't mask the original error
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)